recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+macos_min_version:              # [osx]
+  - 10.7                        # [osx]
```

Comment:
Why are we suddenly supporting osx < 10.9?

Reply:
These are specified in setup.py:
https://github.com/nmslib/nmslib/blob/v2.1.1/python_bindings/setup.py#L132#L134

I think even without the local cbc file, these will applied.

Reply:
so if without cbc they will be applied, why do you need this cbc?

Reply:
For better visibility.

Reply:
so if without cbc they will be applied, why do you need this cbc?

Reply:
Ok...maybe it will spark a question in the future